packages/strapi-plugin-content-type-builder/admin/src/containers/Form/utils/attributeValidations.js

```diff
@@ -36,6 +36,34 @@ export default function checkAttributeValidations(errors) {
     formErrors.push({ name: 'params.key', errors: [{ id: 'content-type-builder.error.attribute.sameKeyAndName' }]});
   }
 
+  const reserved = [
+    'id',
+    'set',
+    'value',
+    'emit',
+    'on',
+    'once',
+    'listeners',
+    'removeListener',
+    'collection',
+    'db',
+    'isModified',
+    'isNew',
+    'get',
+    'modelName',
+    'save',
+    'schema',
+    'toObject',
+    'validate',
+    'remove',
+    '_pres',
+    '_posts',
+  ];
+
+  if (reserved.includes(get(this.props.modifiedDataAttribute, 'name'))) {
```

Comment:
Does Strapi have a includes polyfill because it's not supported in IE11 (https://caniuse.com/#search=includes)
In this case I suggest to use the includes method coming from lodash.